### PR TITLE
Translation Delivery (DEV)

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them.

**Review by mtwalli — DISMISSED** [review]

**Review by harambasicluka — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values-pl/antigen_strings.xml [hunk 1]
```diff
@@ -4,201 +4,205 @@
...
-    <string name="ag_homescreen_card_pcr_clear_test_button">"USUŃ TEST"</string>
+    <string name="ag_homescreen_card_pcr_clear_test_button">"Błąd"</string>
 
     <!-- Body -->
     <!-- XTXT: PCR homescreen card: body - no result -->
-    <string name="ag_homescreen_card_pcr_body_no_result">"Ustalanie wyniku Twojego testu PCR jeszcze się nie zakończyło."</string>
+    <string name="ag_homescreen_card_pcr_body_no_result">"Nie można było wygenerować numeru TAN do przesłania. Skontaktuj się z techniczną infolinią, wybierając opcje: Informacje o aplikacji -&gt; Infolinia techniczna."</string>
     <!-- XTXT: homescreen card: body - negative -->
-    <string name="ag_homescreen_card_pcr_body_result_date">"Test zarejestrowany dnia %1$s"</string>
+    <string name="ag_homescreen_card_pcr_body_result_date">"Wstecz"</string>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values-pl/antigen_strings.xml:84]: This string has wrong format

Comment (harambasicluka) [reply Corona-Warn-App/src/main/res/values-pl/antigen_strings.xml:84]: something went wrong as the whole polish file was changed even if we didn't changed that much, or translation expert will take care of this.

**Review by harambasicluka — CHANGES_REQUESTED** [review]
s. https://github.com/corona-warn-app/cwa-app-android/pull/3465/files#r652644977

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them.

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**